code/game/objects/items/weapons/storage/business_card.dm

```diff
@@ -2,35 +2,82 @@
 	name = "business card holder"
 	desc = "A sleek holster for your business card. You wouldn't want it getting damaged in any way."
 	storage_slots = 5
-	icon = 'icons/obj/office_supplies.dmi'
+	icon = 'icons/obj/bureaucracy.dmi'
 	icon_state = "holder"
 	w_class = ITEMSIZE_TINY
 	max_w_class = ITEMSIZE_TINY
 	can_hold = list(/obj/item/paper/business_card)
 
 /obj/item/storage/business_card_holder/update_icon()
 	cut_overlays()
+	var/overlay_type
 	if(length(contents))
-		var/mutable_appearance/card_overlay = mutable_appearance(icon, "holder-overlay")
+		if(contents.type == /obj/item/paper/business_card/glass) // Could be better, I guess. But it works well enough.
```

Comment:
This part doesn't appear to work, and doesn't take subtypes into account either. Something like this might work out better
```suggestion
		var/obj/O = contents[1]
		if(istype(O, /obj/item/paper/business_card/glass)) // Could be better, I guess. But it works well enough.
```